src/config.ts

```diff
@@ -196,6 +264,16 @@ export const Config = Schema.intersect([
       endpoint: Schema.string().description('API 服务器地址。').required(),
       headers: Schema.dict(String).description('要附加的额外请求头。'),
     }),
+    Schema.object({
+      type: Schema.const('stable-horde'),
+      endpoint: Schema.string().description('API 服务器地址。').default('https://stablehorde.net/'),
+      token: Schema.string().description('授权令牌。').role('secret').default('0000000000'),
```

Comment:
In fact the name of it should be `apikey`, should I update the description?

Reply:
Both have pros and cons, respect your choice.

Reply:
I think it would be better to add a hyper link here which is linked to Stable Horde's documentation. Do you think it is good to be put here or just the document?